src/data-restriction/DataRestrictionUtils.js

```diff
@@ -89,15 +89,15 @@ export function isPrereleaseStudy (access, studyId, user, permissions) {
     if (permissions != null) {
       if (
         access === 'prerelease' &&
-        !isUserFullyApprovedForStudy(permissions, user.properties.approvedStudies, studyId)
+        !isUserFullyApprovedForStudy(permissions, studyId)
       ) {
         return true;
       } else {
         return false;
       }
     }
 
-    if ( (access === 'prerelease') && (!user.properties.approvedStudies.includes(studyId))  ) 
+    if (access === 'prerelease') 
       return true;
     else return false;
```

Comment:
Regarding this block of code:

```typescript
    if (permissions != null) {
      if (
        access === 'prerelease' &&
        !isUserFullyApprovedForStudy(permissions, studyId)
      ) {
        return true;
      } else {
        return false;
      }
    }

    if (access === 'prerelease') 
      return true;
    else return false;
```

It's okay to remove the nil check for `permissions`; that was a guard I had in place when this code was first being rolled out in late 2020, in case the (then-new) dataset access service had a bug that caused the permissions to not be served:

```typescript
    if (
      access === 'prerelease' &&
      !isUserFullyApprovedForStudy(permissions, studyId)
    ) {
      return true;
    } else {
      return false;
    }
```

This block can then be simplified to one line:
```typescript
    return (
      access === 'prerelease' &&
      !isUserFullyApprovedForStudy(permissions, studyId)
    );
```